implementations/apollo-server/metadata.json

```diff
@@ -6,14 +6,17 @@
   "repository": {
     "name": "apollo-server",
     "owner": "apollographql",
+    "maintainer" : "apollographql",
     "link": "https://github.com/apollographql/apollo-server"
   },
   "coreLibrary": {
     "name": "GraphQL.js",
+    "maintainer": "apollographql",
```

Comment:
```suggestion
```
we don't own `GraphQL.js`, we do help with maintenance but I this is misleading

Reply:
thanks, removed